widget/radio.go

```diff
@@ -182,28 +202,30 @@ func (r *Radio) Append(option string) {
 
 // Tapped is called when a pointer tapped event is captured and triggers any change handler
 func (r *Radio) Tapped(event *fyne.PointEvent) {
-	index := 0
-	if r.Horizontal {
-		index = int(math.Floor(float64(event.Position.X) / float64(r.itemWidth())))
-	} else {
-		index = int(math.Floor(float64(event.Position.Y) / float64(r.itemHeight())))
-	}
+	if r.Enabled() {
```

Comment:
Here we can exit early - and if we can we shout.
Use if
```
!r.Enabled() {
    return
}
```
instead - it helps keep code complexity down

Reply:
Yep, totally makes sense. Fixed, pushed.